support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -47,39 +74,49 @@ import { CheckoutDivider } from 'pages/supporter-plus-landing/components/checkou
 import { GuardianTsAndCs } from 'pages/supporter-plus-landing/components/guardianTsAndCs';
 
 /** App config - this is config that should persist throughout the app */
-const countryGroupIds = ['uk', 'us', 'eu', 'au', 'nz', 'ca', 'int'] as const;
-const CountryGroupIdSchema = picklist(countryGroupIds);
-const countryGroupId = parse(
-	CountryGroupIdSchema,
-	window.location.pathname.split('/')[1],
-);
+const isTestUser = true;
+const geoIds = ['uk', 'us', 'eu', 'au', 'nz', 'ca', 'int'] as const;
+const GeoIdSchema = picklist(geoIds);
+const geoId = parse(GeoIdSchema, window.location.pathname.split('/')[1]);
 
 let currentCurrency: Currency;
 let currentCurrencyKey: keyof typeof currencies;
-switch (countryGroupId) {
+let countryGroupId: CountryGroupId;
```

Comment:
I've gone with `geo` and `countryGroup`, although we should look at syncing this up with the server side, which I think it clearer, and uses the URL part and the `countryGroupId`.